packages/moonstone/MoonstoneDecorator/MoonstoneDecorator.js

```diff
@@ -58,10 +59,14 @@ const defaultConfig = {
  * @public
  */
 const MoonstoneDecorator = hoc(defaultConfig, (config, Wrapped) => {
-	const {ri, i18n, spotlight, float, noAutoFocus, overlay, textSize, skin, highContrast} = config;
+	const {ri, i18n, spotlight, float, noAutoFocus, overlay,
+		textSize, skin, highContrast, disableFullscreen} = config;
 
 	// Apply classes depending on screen type (overlay / fullscreen)
-	const bgClassName = 'enact-fit' + (overlay ? '' : ` ${css.bg}`);
+	const bgClassName = classNames({
+		'enact-fit': !disableFullscreen,
+		[`${css.bg}`]: !overlay
```

Comment:
I believe it's possible to reduce this from `[\`${css.bg}\`]` to simply `bg`. I may be confusing this with features of our `kind()` though.

Reply:
tried, but failed...